evalai/utils/common.py

```diff
@@ -34,3 +35,25 @@ def validate_token(response):
         if (response['detail'] == 'Token has expired'):
             echo(style("\nSorry, the token has expired. Please generate it again.\n", bold=True, bg="red"))
             sys.exit(1)
+
+
+def validate_date_format(date):
+    for fmt in ("%Y-%m-%dT%H:%M:%S.%fZ", "%Y-%m-%dT%H:%M:%SZ"):
+        try:
+            return datetime.strptime(date, fmt)
+        except ValueError:
+            pass
```

Comment:
I think we should also an error when an exception occurs `raise ValueError('Invalid date format. Please check again!')`

Reply:
No, so the loop would terminate and it would only check the first format